File: fib.rb
#!/usr/bin/ruby

def fib n
    if n == 0 || n == 1
        return n
    else
        return fib(n - 2) + fib(n - 1)
    end
end

n = ARGV.shift.to_i
puts "%s, %s" % [n, fib(n)]

File: fib.py
#!/usr/bin/python

import sys

def fib(n):

    if n == 0 or n == 1:
        return n

    return fib(n - 1) + fib(n - 2)

if __name__ == "__main__":

    n = int(sys.argv[1])
    print "%d, %d" % (n, fib(n))
